test/suite/telemetry.test.ts

```diff
@@ -18,32 +18,6 @@ suite('GATelemetry', function () {
   teardown(() => {
     sandbox.restore();
   });
-
```

Comment:
Looks like we could remove the entire suite starting from line 10

Reply:
You're right! Removing